lib/mpl_toolkits/axes_grid1/axes_rgb.py

```diff
@@ -89,12 +96,12 @@ def __init__(self, *args, pad=0, **kwargs):
         ----------
         pad : float, default: 0
             fraction of the axes height to put as padding.
-        axes_class : matplotlib.axes.Axes
-
+        axes_class : `~matplotlib.axes.Axes`
+            Axes class to use. If not provided ``_defaultAxesClass`` is used.
```

Comment:
`_defaultAxesClass` is not documented and private; is it different in different places? Can we document this in some way that is clearer?

Reply:
There is https://github.com/matplotlib/matplotlib/blob/aaa5129a9fb472a239bb7421635c31f20a523c5f/lib/mpl_toolkits/axes_grid1/axes_rgb.py#L70

Reply:
Hmm, yes, but that doesn't really say what it is in docs then. For example, this one uses `Axes`, but that's not the standard one, but the one in `axes_grid1`.

Reply:
I've added a line in both `RGBAxes` docstrings stating which the `_defaultAxesClass` is.